build/azDevOps/azure/air-infrastructure-data.yml

```diff
@@ -106,154 +106,101 @@ stages:
                     TF_VAR_name_domain: $(domain)
                     TF_VAR_stage: $(Environment.ShortName)
 
-  # yamllint disable rule:comments-indentation
-  # - stage: Prod
-  #   dependsOn: Build
-  #   condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/main'))
-  #   variables:
-  #     - group: amido-stacks-infra-credentials-prod
-  #     - group: stacks-credentials-prod-kv
-  #     - name: Environment.ShortName
-  #       value: prod
-  #     - name: dns_base_domain # Available in Terraform Output, but not yet enabled to export as pipeline vars
-  #       value: prod.amidostacks.com
-  #     - name: core_environment
-  #       value: prod
-  #   jobs:
-  #     - deployment: AppInfraProd
-  #       pool:
-  #         vmImage: $(pool_vm_image)
-
-  #       environment: ${{ variables.domain }}-prod
-
-  #       strategy:
-  #         runOnce:
-  #           deploy:
-  #             steps:
+#   yamllint disable rule:comments-indentation
```

Comment:
If you just put the comment on the line before this shouldn't be a problem I think?

Reply:
udated